chalk-ir/src/lib.rs

```diff
@@ -1803,8 +1802,8 @@ impl<T: HasInterner> Binders<T> {
     /// Wraps the given value in a binder without variables, i.e. `for<>
     /// (value)`. Since our deBruijn indices count binders, not variables, this
     /// is sometimes useful.
-    pub fn empty(interner: &T::Interner, value: T) -> Self {
-        let binders = VariableKinds::new(interner);
+    pub fn without_vars(interner: &T::Interner, value: T) -> Self {
```

Comment:
I think I actually like `empty` here. Or `dummy` if we want to match rustc.

Reply:
I agree that `without_vars` wasn't very clear to me.